assets/css/v2/bus_eink/simulation.scss

```diff
@@ -22,23 +23,41 @@
     display: flex;
     margin-top: auto;
     margin-bottom: auto;
-    border-radius: 8px;
+    height: 316px;
+    border-radius: 0 8px 8px 0;
+    position: relative;
+
+    &:before {
+      content: "";
+      position: absolute;
+      right: 100%;
+      height: 216px;
+      border-top: 50px solid transparent;
+      border-right: 70px solid #2e3f4d;
+      border-bottom: 50px solid transparent;
+    }
```

Comment:
A more systematic way of creating this triangle is taking the height of the projection (316px), dividing it by two, and setting the top and bottom borders to that value, with a height of `0px`. The border-right value will be how steep you want the triangle. Keep content, position, and right as-is. So for e-ink:
```
&:before {
    content: "";
    right: 100%;
    // NO HEIGHT
    border-top: 158px solid transparent;
    border-right: 128px solid #2e3f4d;
    border-bottom: 158px solid transparent;
}
```
 Borrowed from https://css-tricks.com/snippets/css/css-triangle/.

Reply:
For PreFare as another example, it looks like border-right would be approx `144px` and top/bottom would be `72px`.

Reply:
Awesome! SO much better.